src/lib/mailerliteClient.ts

```diff
@@ -0,0 +1,29 @@
+import MailerLite from '@mailerlite/mailerlite-nodejs'
+
+const mailerliteInstance = new MailerLite({
+  api_key: process.env.NEXT_PUBLIC_MAILERLITE_KEY || '',
```

Comment:
**issue:** very important! it cannot be NEXT_PUBLIC as its exposed to the client then. Just do `MAILERLITE_API_KEY`